ARExperiment/ARExperiment/OneModelUsingAnchorsViewController.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+import SceneKit
+import ARKit
+
+class OneModelUsingAnchorsViewController: UIViewController {
+
+    @IBOutlet var sceneView: ARSCNView!
+    //MARK: These strings is what you need to switch between different 3D objects
+    /** NodeName is the name of the object you want to show, not necessarily the name of the file.
```

Comment:
Is it worth putting all this in a README?

Reply:
I alreday created the Readme, is the next PR 😄